etcd-manager/cmd/etcd-manager/main.go

```diff
@@ -189,6 +196,32 @@ func (o *EtcdManagerOptions) InitDefaults() {
 	o.Insecure = false
 	o.EtcdInsecure = false
 	o.EtcdManagerMetricsPort = 0
+
+	o.NetworkCIDR = os.Getenv("ETCD_MANAGER_NETWORK_CIDR")
```

Comment:
Not quite sure we want this.

Reply:
This is just to make it easier to configure it within kOps, because currently the API config only exposes `env`, but no way to add parameters:
* https://pkg.go.dev/k8s.io/kops/pkg/apis/kops#EtcdManagerSpec

But probably it's possible to do scrap the ENV var and I need to find another way to configure this by either add a possibility to modify the parameters in the kOps API or some other supported way (by means of cloud-init, etc.).